ui/e2e/explorer/workflow/instance.spec.ts

```diff
@@ -0,0 +1,86 @@
+import {
+  createInstance,
+  simpleWorkflow as simpleWorkflowContent,
+} from "./utils";
+import { createNamespace, deleteNamespace } from "../../utils/namespace";
+import { expect, test } from "@playwright/test";
+
+import { createFile } from "e2e/utils/files";
+import { faker } from "@faker-js/faker";
+
+let namespace = "";
+const simpleWorkflowName = faker.system.commonFileName("yaml");
+
+test.beforeEach(async () => {
+  namespace = await createNamespace();
+  /* create workflows we can use to create instances later */
+  await createFile({
+    name: simpleWorkflowName,
+    namespace,
+    type: "workflow",
+    yaml: simpleWorkflowContent,
+  });
+});
+
+test.afterEach(async () => {
+  await deleteNamespace(namespace);
+  namespace = "";
+});
+
+test("the header of the instance page shows the relevant data for the workflow", async ({
+  page,
+}) => {
+  const newInstance = createInstance({ namespace, path: simpleWorkflowName });
+
+  await expect(newInstance, "wait until process was completed").toBeDefined();
+
+  const instanceID = (await newInstance).instance;
+
+  await page.goto(`/${namespace}/instances/${instanceID}`);
+
+  const header = page.getByTestId("instance-header-container");
+  await expect(header, "the header is visible").toBeVisible();
+
+  const instanceID_Header = header.locator("h3");
+  await expect(instanceID_Header, "the instance ID is visible").toBeVisible();
+
+  const instanceID_navLink = page.locator("ul").locator("a").nth(2);
+
+  expect(instanceID_Header.innerText, "the instance IDs are the same").toEqual(
```

Comment:
Same here about the description, "the instance IDs are the same" could be "It renders the instance ID in the breadcrumb navigation"

Reply:
I disagree with renaming this test because it really is what it says: a check if the IDs are equal.
I want to keep this test because I think that it is important too. 
But I added another test which checks what you meant. 🙂